fdm-app/app/routes/farm.$b_id_farm.field.$b_id.overview.tsx

```
@@ -0,0 +1,367 @@
+    const form = useRemixForm<z.infer<typeof FormSchema>>({
+        mode: "onTouched",
+        resolver: zodResolver(FormSchema),
+        defaultValues: {
+            b_name: loaderData.field.b_name,
+            b_acquiring_method: loaderData.field.b_acquiring_method,
+            b_acquiring_date: loaderData.field.b_acquiring_date,
+            b_terminating_date: loaderData.field.b_terminating_date,
+        },
+    })
+
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: form.reset

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)